zuul.yaml

```diff
@@ -2,8 +2,8 @@
     check:
       jobs:
         - wazo-acceptance-job:
-            nodeset: debian10-vm
+            nodeset: debian10-vm-m2
     gate:
       jobs:
         - wazo-acceptance-job:
-            nodeset: debian10-vm
+            nodeset: debian10-vm-m2
```

Comment:
Do we really need `-m2` with all optimization done on linphone docker?

Reply:
Currently the VM have 2G and only between 250Mo/300Mo free RAM when everything is installed.

Reply:
And we still see that job fail with out of memory issue